langchain/text_splitter.py

```diff
@@ -478,6 +478,45 @@ def __init__(self, **kwargs: Any):
         super().__init__(separators=separators, **kwargs)
 
 
+class HtmlTextSplitter(RecursiveCharacterTextSplitter):
```

Comment:
This feels like a splitter that's similar conceptually to the code language splitters below (and to markdown, rst).

And it feels very different from SpacyTextSplitter or NLTKTextSplitter.

I don't have a strong opinion here, mostly wondering if it makes sense to keep a small number of core classes in the global namespace, and do the parameterization as part of the initializer when makes sense